c++/src/kj/async-io-unix.c++

```diff
@@ -1483,7 +1483,7 @@ kj::Own<PeerIdentity> SocketAddress::getIdentity(kj::LowLevelAsyncIoProvider& ll
         result.uid = creds.uid;
       }
 
-#elif defined(LOCAL_PEERCRED)
+#elif defined(LOCAL_PEERCRED) && __OpenBSD__
```

Comment:
It looks like your change on this line will actually break platforms other than OpenBSD that use this block of code.

Reply:
This is a mistake and is now reverted, thanks!